### ISSUE TITLE
Could you please explain the mathematical principle about the calculation of extremumLat?

### ISSUE THREAD

**Issue by DragonEmperorG** [Shrulik/NGeoKDBush#1]
![image](https://user-images.githubusercontent.com/24807062/50052061-e60ef300-0158-11e9-84fe-123792b379e3.png)
Could you please offer me some papers or related material to prove the BoxDist calculation？
I have full understood the calculation of the great circle distance, but couldn't figure it out that when the point is within this range, why the 
`var extremumLat = Math.Atan(sinLat / (cosLat * cosLngDelta)) / rad;` 
but not 
`var extremumLat = Math.Atan(sinLat / cosLat) / rad;`

**Comment by Shrulik** [comment]
For a complete answer you would have to ask @mourner at https://github.com/mourner/geokdbush from which I ported this code. I tested this code for correctness,  but I'm not even sure if I ever fully understood it or just forgot. 

I don't have time to look into it seriously at the moment, I think my entry point to understanding the algorithms was this article :  https://blog.mapbox.com/a-dive-into-spatial-search-algorithms-ebd0c5e39d2a
(  also by mourner ).
